google_nest_sdm/camera_traits.py

```diff
@@ -278,6 +279,12 @@ async def generate_web_rtc_stream(self, offer_sdp: str) -> WebRtcStream:
         results = response_data[RESULTS]
         obj = WebRtcStream.from_dict(results)
         obj._cmd = self.cmd
+        _LOGGER.debug("Received answer_sdp: %s", obj.answer_sdp)
+        if "mozilla" in offer_sdp:
```

Comment:
After seeing the code coverage report, i'm thinking maybe this line should move into `fix_mozilla_sdp_answer` (then always call it) and it will be easy to add a test to get 100% coverage.